go.mod

```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
 
 go 1.20
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace (
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/channelserver v0.5.1-0.20230719220800-0a37b73c7df8 => github.com/KevinJoiner/channelserver v0.0.0-20240124134223-b6d3819b0a98
```

Comment:
Remove and update commit after https://github.com/rancher/channelserver/pull/25 is merged.